numba_dpex/core/kernel_interface/dispatcher.py

```diff
@@ -586,6 +614,20 @@ def _check_ranges(self, device):
                 device=device,
             )
 
+    def _strip_usm_metadata(self, argtypes):
+        stripped_argtypes = []
+        for argty in argtypes:
+            if isinstance(argty, USMNdArray):
+                # Convert the USMNdArray to an abridged type that disregards the
```

Comment:
Might as well move this comment into function docstring.

Reply:
Done